editor/filesystem_dock.cpp

```diff
@@ -1256,6 +1256,9 @@ void FileSystemDock::_update_resource_paths_after_move(const Map<String, String>
 		}
 
 		r->set_path(base_path + extra_path);
+		if (InspectorDock::get_inspector_singleton()->get_edited_object() == r.ptr()) {
```

Comment:
This does not seem like the right fix. Usually, when an edited object is modified, the inspector automatically detects the changes done to a modified property. Thus the change of the `path` property should be detected by the Inspector, and it should automatically trigger an update.

As a consequence, the change should likely be made in the InspectorDock, not here.

Reply:
Done.
Updating of `InspectorDock` was made in correspondence to `EditorInspector`'s mechanism

Reply:
I think polling the path like this PR does currently is an overkill. Maybe we could add a `file_renamed` signal to FileSystemDock and connect it in the inspector.